addons/snailmail/static/src/components/snailmail_notification_popover/snailmail_notification_popover.xml

```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <t t-name="snailmail.SnailmailNotificationPopover" owl="1">
-        <t t-if="message">
-            <div class="o_SnailmailNotificationPopover" t-attf-class="{{ className }}" t-ref="root">
-                <t t-if="notification">
-                    <i class="o_SnailmailNotificationPopover_icon" t-att-class="iconClass" role="img"/>
-                    <span t-esc="iconTitle"/>
-                </t>
-            </div>
-        </t>
+    <t t-name="snailmail.SnailmailNotificationPopoverContent" owl="1">
+        <div class="o_SnailmailNotificationPopoverContent m-2" t-attf-class="{{ className }}" t-ref="root">
+            <t t-if="notification">
```

Comment:
I'm not sure if this conditions is necessary. It feels like the popover should not even exist if there is no notification to display